corehq/apps/data_cleaning/templatetags/data_cleaning.py

```diff
@@ -45,3 +46,44 @@ def dc_data_type_icon(data_type):
             context
         )
     )
+
+
+@register.simple_tag
+def edited_value(record, bound_column):
+    """
+    Returns the Edited value of a record based on
+    the `BoundColumn` information.
+
+    We return `Ellipsis` if the value is not set in the `edited_properties`
+    dictionary. This is because the original value can be made `None`
+    by the cleaning actions, and we want to be able to reflect that value.
+
+    :params record:
+        EditableCaseSearchElasticRecord instance
+    :params bound_column:
+        BoundColumn instance
+    :returns:
+        The edited value of the record based on the
+        `BoundColumn` information.
+    :rtype:
+        str | Ellipsis
+    """
+    return record.edited_properties.get(bound_column.name, Ellipsis)
```

Comment:
the purpose of this `Ellipsis` is to represent the concept of "no value". Can we create a separate variable and assign `Ellipsis` to that, so this reads like `.get(bound_column.name, NO_VALUE)` instead? it makes it slightly more intuitive to follow

Reply:
https://github.com/dimagi/commcare-hq/pull/36247/commits/34599aa50425dff64ae52d845c9b10367a325f55